bigbluebutton-html5/public/locales/en.json

```diff
@@ -198,14 +198,14 @@
     "app.userList.menu.lockPublicChat.label": "Lock public chat",
     "app.userList.menu.unlockPublicChat.label": "Unlock public chat",
     "app.userList.userOptions.manageUsersLabel": "Manage users",
-    "app.userList.userOptions.muteAllLabel": "Mute all users",
-    "app.userList.userOptions.muteAllDesc": "Mutes all users in the session",
+    "app.userList.userOptions.usersJoinMutedEnableLabel": "Enable Users join muted",
```

Comment:
If you have any viewers with open mic, pressing this also mutes them (at least on 2.7). I find this new label wording confusing. If you want to be more precise perhaps use "Mute all users (includes ones joining later)"...

Reply:
cc @prlanzarin as this is touching on `muteOnStart` which I know is a topic of interest.